Mathlib/Topology/Category/TopCat/Limits/Pullbacks.lean

```diff
@@ -52,9 +52,8 @@ def pullbackCone (f : X ⟶ Z) (g : Y ⟶ Z) : PullbackCone f g :=
     (by
       dsimp [pullbackFst, pullbackSnd, Function.comp]
       ext ⟨x, h⟩
-      simp only [TopCat.comp_app]
-      simp only [hom_apply]
-      simp [h, ← hom_apply])
+      rw [comp_apply, ContinuousMap.coe_mk, comp_apply, ContinuousMap.coe_mk]
+      exact h)
```

Comment:
Are we missing here a version of `ContinuousMap.coe_mk` that refers to the new FunLike instance? I would like to just be able to invoke it via `simp` here. (and the next few proofs).

Reply:
Oh, the bigger problem is that you've had to write `exact h` instead of `simp [h]` because of the competing `FunLike` instances.

Reply:
(Ah, as you explained at https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/coe.20conflict.20in.20concrete.20categories/near/363591060. Sorry to be slow.)